api/profile/src/utils.rs

```diff
@@ -0,0 +1,57 @@
+use std::collections::HashMap;
```

Comment:
nit: we don't currently have a convention for `utils.rs` (it's surprisingly a delicate subject 😆 ). I would suggest is moving the contents of this file in `lib.rs`.

Reply:
Gotcha, I was trying to keep the lib.rs file solely focused on HTTP handling but I'll move them into the lib file if that is conventional for Rust.